src/things/mod.rs

```diff
@@ -89,20 +89,24 @@ impl FromStr for ThingType {
     }
 }
 
+pub type SubredditSet = ShredditSet;
+pub type CommentIdSet = ShredditSet;
+pub type PostIdSet = ShredditSet;
+
 #[derive(Debug, Clone)]
-pub struct SubredditSet(HashSet<String>);
+pub struct ShredditSet(HashSet<String>);
 
-impl std::convert::From<&str> for SubredditSet {
+impl std::convert::From<&str> for ShredditSet {
     fn from(s: &str) -> Self {
         let mut subreddits = HashSet::<String>::new();
         s.split(',').for_each(|f| {
             subreddits.insert(f.to_owned());
         });
-        SubredditSet(subreddits)
+        ShredditSet(subreddits)
```

Comment:
i know it's from a previous PR, but i must have missed this. it's a nit, but IMO it would be nicer to do something like `Self(s.split(',').map(Into::into).collect())`. no mutability involved and a one liner!

Reply:
wow i love it, updating